### ISSUE TITLE
UIA navigation doesn't work with the TabControl

### ISSUE THREAD

**Issue by Vino-Wang** [dotnet/winforms#3058]
**.NET Core Version:**
.NET SDK 5.0.100-Preview.4.20209.25 + latest VS2019

**Have you experienced this same bug with .NET Framework?:**
Yes

**Problem description:**
UIA navigation to move backwards/forwards through AIWin(Accessibility Insights For Windows) keyboard combination Shift+Ctrl+F5/F6/F7/F8/F9 doesn't work with the TabControl. AIWin can’t move backwards/forwards correctly through the set of sibling tab items. This issue also can reproduce when targeting .NET Core 3.1.

Please see the attached repro video: 
[AIWinKeyboard_function_issue_with_Winforms TabControl.zip](https://github.com/dotnet/winforms/files/4473414/AIWinKeyboard_function_issue_with_Winforms.TabControl.zip)

**Expected behavior:**
AIWin should be able to move backwards/forwards through the tab items.
Please see the attached repro video:
[NotReproWithOtherElements.zip](https://github.com/dotnet/winforms/files/4473417/NotReproWithOtherElements.zip)

**Minimal repro:**
1.	Build a new .NET Core WinForms app with a tab control that have a few tab items. Or open the attached app. 
[WinFormsTabTest_AIWinKeyboardIssue.zip](https://github.com/dotnet/winforms/files/4473408/WinFormsTabTest_AIWinKeyboardIssue.zip)
2.	Run the app, and launch the Accessibility Insights For Windows (AIWin) tool at it. Verify that the UIA hierarchy seems reasonable. 
3.	Use the AIWin keyboard shortcuts of Shift+Ctrl+F5/F6/F7/F8/F9 to move AIWin around the UIA tree particular ‘tab’ tree. For more details about the shortcuts behavior please visit https://accessibilityinsights.io/docs/en/windows/reference/keyboard.

**Comment by SergeySmirnov-Akvelon** [comment]
![image](https://user-images.githubusercontent.com/23376742/126775868-fb191976-8b7e-40ee-989b-421cc681b926.png)

![3058-Prototype](https://user-images.githubusercontent.com/23376742/126778993-bbfddfa2-7222-42d0-8cfc-c0d3cbe3b560.gif)

**Comment by Philip-Wang01** [comment]
Verified this issue with .NET 6.0 RC2 latest build: 6.0.10-rc.2.21465.13. UIA navigation is suitable for TabControl
![image](https://user-images.githubusercontent.com/86937911/133567152-216ee11b-5820-4368-b85b-53013a4d6709.png)

**Comment by Jenna-Zheng** [comment]
Verified this issue with latest .NET 7.0.100-alpha.1.21479.21 build, it is fixed.
![image](https://user-images.githubusercontent.com/64949616/135396854-4dc28756-b8c1-4c96-8fec-538950bff0fc.png)
![Issue_TestUIA](https://user-images.githubusercontent.com/64949616/135398107-958a374e-6c23-49f6-a513-81fc0021fe6c.gif)
